docs/features/instance-filtering.md

```
@@ -72,6 +72,13 @@ Alternatively, pass `groupBy` and `aggregations` as query parameters:
 ?filter={"status":{"eq":"Active"}}&groupBy={"field":"attributes.status"}&aggregations={"count":true}
 ```
 
+When `groupBy`/`aggregations` are combined with GraphQL filter, instance columns in `filter` (for example `createdAt`, `modifiedAt`, `completedAt`, `status`, `key`) are applied together with JSON attribute filters.
+Example:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)